docs/monitoring/inventory.md

```diff
@@ -1,13 +1,107 @@
 title: Sematext Inventory Monitoring
 description: Sematext Inventory Monitoring gives you insight into your whole infrastructure automatically based on the applications, containers, servers and general infrastructure you are monitoring.
 
-This should explain the new Inventory Monitoring feature, and have the following sections.
+##Overview
```

Comment:
You can remove this heading altogether. It makes sense from the `title` what the paragraph beneath is about.

Reply:
OK, done.